src/GitLabApiClient/Internal/Queries/GroupsProjectsQueryBuilder.cs

```diff
@@ -43,18 +46,20 @@ private static string GetVisibilityQueryValue(GroupsVisibility visibility)
                 case GroupsVisibility.Private:
                     return "private";
                 default:
-                    return "public";
```

Comment:
if we are removing public as default we will need to add Public condition

Reply:
public condition is already added in the switch statement.